task2_01

ShakhriyorToylokov1999 opened the pull request.

ShakhriyorToylokov1999 committed:
task2_01

Mirazyz commented on Task2_01/Form1.cs:
```diff
@@ -0,0 +1,10 @@
+{
+    public partial class Form1 : Form
+    {
+        public Form1()
+        {
+            InitializeComponent();
```
Improve this lines of codes

ShakhriyorToylokov1999 committed:
Task 1.2:
.NET Framework – WinForms/WPF
Application requirements:

Input: {username} (for console app – as a command line parameter)

Output: “Hello, {username}” (via form or separate window)
